app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.directive.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.pipeline.config.triggers.travis.options.directive', [
+    require('core/ci/igor.service.js')
+  ])
+  .directive('travisTriggerOptions', function () {
```

Comment:
I am not sure if the corresponding `jenkinsTriggerOptions` is a directive, but it would be preferable if this were a component (it doesn't look like it needs to be a directive). 

The syntax:

```
.component('travisTriggerOptions`, { 
  templateUrl: ...,
  bindings: { command: '='},
  controller: ...
})
```

`$ctrl` is the default value for `controllerAs` for a component.